orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -72,6 +73,19 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
   }
 
   @Override
+  long getExponentialBackoffPeriod(Duration taskDuration) {
```

Comment:
This is the only hierarchy of task that actually specifies a dynamic backoff period ... encompasses most of the WaitForXXX instance tasks.